src/components/AddNewModal.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from './css/modal.css';
+
+/**
+* Component for Modal Dialog.
+* Contains the form that allows users to add a new publication to the data.
+**/
+
+export default class AddNewModal extends React.Component {
```

Comment:
I think this should simply be:
```
class AddNewModal extends React.Component {
```
Then, at the bottom of the file:
```
module.exports = AddNewModal;
```

Reply:
The export class syntax can be used while defining if there is only one class in the file. I can change it to module.exports if that is the standard way.

Reply:
Yes, our client uses module.exports so let's use that too.